crates/primitives/src/transaction/pooled.rs

```diff
@@ -70,7 +62,8 @@ impl PooledTransactionsElement {
         if tx.is_eip4844() {
             return Err(tx)
         }
```

Comment:
I notice that the docs here are a bit ambiguous, this conversion is specifically for incoming broadcasts:

https://github.com/paradigmxyz/reth/blob/abab5a6938e0a4318f1bda9b398560ed6cc4a91e/crates/net/network/src/transactions/mod.rs#L854-L864

so we should mention that in the docs that this is intended to be used as a helper to convert decoded p2p tx broadcast messages